.gitlab-ci.yml

```diff
@@ -18,3 +24,32 @@ test:unit:
     - git submodule update --init --recursive
   script:
     - make test
+
+test:integration:
+  stage: test
+  needs: []
+  image: docker:18-dind
+  tags:
+    - mender-qa-slave
+  before_script:
+    - /usr/local/bin/dockerd-entrypoint.sh &
+    - sleep 10
+    - export DOCKER_HOST="unix:///var/run/docker.sock"
+    - docker version
+    - export
```

Comment:
For a fresh new repo, please let's start already with `docker:19`. See this PR as a reference for what needs to be done: https://github.com/mendersoftware/mender-demo-artifact/pull/80

Reply:
Done